boot/zephyr/include/sysflash/pm_sysflash.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef __PM_SYSFLASH_H__
+#define __PM_SYSFLASH_H__
+/* Blocking the __SYSFLASH_H__ */
+#define __SYSFLASH_H__
+
+#include <pm_config.h>
+#include <mcuboot_config/mcuboot_config.h>
+
+#ifndef CONFIG_SINGLE_APPLICATION_SLOT
+
+#if (MCUBOOT_IMAGE_NUMBER == 1)
+
+#define FLASH_AREA_IMAGE_PRIMARY(x)     PM_MCUBOOT_PRIMARY_ID
+#define FLASH_AREA_IMAGE_SECONDARY(x)   PM_MCUBOOT_SECONDARY_ID
+
+#elif (MCUBOOT_IMAGE_NUMBER == 2)
```

Comment:
>= 2 right? Either that or needs an #error for > 2 because of the third image XIP?

Reply:
I have just move code here, I have not applied any more changes than required for the code to be moved.
This line comes in original form from previously modified sysflash.h

Reply:
I guess this needs to be taken up in a separate issue. I would create a task for it if this seems needed.